.github/workflows/ecr_gc_docker.yml

```diff
@@ -0,0 +1,40 @@
+name: Garbage Collect ECR Images
```

Comment:
TIL nice usage of ghstack to dry run! BTW, the concurrency needs to be updated to pass the check